experimental/embeddings/scripts/evaluate_document_classification.py

```diff
@@ -4,38 +4,41 @@
 import typer
```

Comment:
do we want to add `sequence tagging` evaluation in this PR as well?

Reply:
Yes, `sequence tagging` will be also added.